alerta/auth/__init__.py

```diff
@@ -1,14 +1,19 @@
-
 from flask import Blueprint, request
 
 from alerta.exceptions import ApiError
 
-auth = Blueprint('auth', __name__)
-
-from . import basic, github, gitlab, google, keycloak, pingfederate, saml2, userinfo
+auth = Blueprint('auth', __name__) 
 
+def init_app(app):
```

Comment:
As with change in alerta/app.py: 
Unsure if this is the best way, however I had issues with reading the configuration any other way.
Issue occurs as the Flask app configuration seems not to be available at import-time.